packages/akashic-cli-export/src/zip/licenseUtil.ts

```diff
@@ -58,3 +68,10 @@ async function makeLicenseInfo(source: string, pkgJsonPaths: string[]): Promise<
     };
     return licenseInfos;
 }
+
+function validateLicense(fileName: string, libName: string, license: string): void {
+    // MIT/ISC 以外のライセンスは警告
+    if (!/(MIT|ISC)/i.test(license)) {
+        console.warn(`[WARNING]: LICENSE for ${libName} is "${license}".`);
```

Comment:
エラー (or 警告) メッセージは「何が悪いのか」「どうすればいいのか」わかるようにしたいです。こんな塩梅でどうでしょうか。

```
[WARNING]: The license of <libName> will not be included in thirdparty_license.txt since akashic export doesn't know its license "<license>". You may need to follow the license by yourself.
```

Reply:
ありがとうございます。警告メッセージ修正しました。メッセージに合わせ、ファイル名、ライセンス種別で警告を出力するものは、`thirdparty_license.txt` に含めないよう修正しました。